lib/evmone/instructions.hpp

```diff
@@ -783,6 +783,57 @@ inline void gas(StackTop stack, ExecutionState& state) noexcept
     stack.push(state.gas_left);
 }
 
+namespace
+{
+
+template <size_t Len>
+uint64_t load(code_iterator pos) noexcept
```

Comment:
```suggestion
inline uint64_t load(code_iterator pos) noexcept
```

I'd also call the function `load_partial_push_data()`.